public/javascripts/Gallery/css/gallery.css

```diff
@@ -1,3 +1,201 @@
+.gallery-card {
```

Comment:
@rpechuk why is there all this added CSS?

Reply:
That's super weird i'm not sure, let me role back the css and then put in just what I changed.

Reply:
@misaugstad should be fixed now 👍